includes/admin.form.inc

```diff
@@ -34,13 +34,22 @@ function islandora_xmlsitemap_admin_form($form, &$form_state) {
     '#description' => t('Solr field in which we can perform sorting and range queries.'),
     '#default_value' => variable_get('islandora_xmlsitemap_last_modified_field', 'fgs_lastModifiedDate_dt'),
   );
-  $form['islandora_xmlsitemap_number_of_pids_to_process'] = array(
+  $form['islandora_xmlsitemap_generate_limit_cron'] = array(
```

Comment:
@janjouketjalsma If you are going to change this variable name, you should add a [`hook_update_N()`](https://api.drupal.org/api/drupal/modules%21system%21system.api.php/function/hook_update_N/7.x) function to migrate the old setting from `islandora_xmlsitemap_number_of_pids_to_process` to `islandora_xmlsitemap_generate_limit_cron` and then unset the `islandora_xmlsitemap_number_of_pids_to_process` variable. Otherwise it will hang around forever.

Reply:
Once again, the old refrain, "Don't forget the uninstall hook!" 
islandora_xmlsitemap_uninstall() - the variable name should be updated there too.